### PR TITLE
Ci relnotes 1.28 ci 12644

### PR DESCRIPTION
**Pull request by douglas-j-bothwell**
### Preview link

- Continuous Integration release notes > [Version 1.28.2](https://665095d9c6fdfe009c887f01--harness-developer.netlify.app/release-notes/continuous-integration#version-1282)

### Merge requirements

PRs must meet these requirements to be merged:

- [x] Successful preview build.
- [x] Code owner review.
- [x] No merge conflicts.
- [x] Release notes/new features docs: Feature/version released to at least one prod environment.

### CODE REVIEW

File: release-notes/continuous-integration.md [hunk 1]
```diff
@@ -36,6 +36,23 @@ Contact [Harness Support](mailto:support@harness.io) if you have any questions.
...
 ## May 2024
 
+### Version 1.28.2
+
+<!-- 21 May 2024 -->
+
+#### New features
+
+- This release adds support for CI resource classes, which you can use to allocate the build machines you want to use in Harness Cloud. (CI-10509)
+
```
Comment (brian-f-harness) [inline release-notes/continuous-integration.md:46]: CI-12130 was a delegate-dependent item released today. Normally we include these items in the module release notes as well with the standard verbiage about the required delegate version.
https://developer.harness.io/release-notes/delegate#version-240583001-

**Review by brian-f-harness — APPROVED** [review]
Added a comment for a delegate dependent item that isn't listed, but otherwise lgtm

File: release-notes/continuous-integration.md [hunk 2]
```diff
@@ -36,6 +36,25 @@ Contact [Harness Support](mailto:support@harness.io) if you have any questions.
...
+  This item requires Harness Delegate version 24.05.83001. For information about Harness Delegate features that require a specific delegate version, go to the [Delegate release notes](/release-notes/delegate).
```
Comment (brian-f-harness) [inline release-notes/continuous-integration.md:47]:
```suggestion
  - Added support for proxies via Secure Connect for GitHub App connectors. (CI-12130, ZD-61883)
  
  This item requires Harness Delegate version 24.05.83001. For information about Harness Delegate features that require a specific delegate version, go to the [Delegate release notes](/release-notes/delegate).
```

**Review by brian-f-harness — APPROVED** [review]

**Comment by douglas-j-bothwell** [comment]
Reviewed/approved via zoom by @nofarblue

**Pull request merged**